### PR TITLE
Add test for causal manager None categoricals

### PR DESCRIPTION
**Pull request by gregorybchris**

### CODE REVIEW

**Review by gaugup — APPROVED** [review]

File: responsibleai/responsibleai/_managers/causal_manager.py [hunk 1]
```diff
@@ -225,9 +225,13 @@ def compute(self):
             y = pd.concat([self._train, self._test], ignore_index=True)[
                 self._target_column].values.ravel()
 
+            categoricals = self._categorical_features
+            if categoricals is None:
+                categoricals = []
```
Comment (gaugup) [inline responsibleai/responsibleai/_managers/causal_manager.py:230]: Could you possibly add a comment saying that this is a limitation in econml and would be addressed by econml?

Comment (gaugup) [reply responsibleai/responsibleai/_managers/causal_manager.py:230]: Would add this in a follow up PR.

**Review by imatiach-msft — APPROVED** [review]

**Review by xuke444 — APPROVED** [review]
